src/components/DeviceSummaryTile.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {
+  Card,
+  CardTitle,
+  CardBody,
+  Button,
+  Bullseye,
+  Spinner,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { useSelector, shallowEqual } from 'react-redux';
+
+const DeviceSummaryTileBase = ({
+  orphaned,
+  active,
+  no_reports,
+  never_reported,
```

Comment:
No need to use snake case in here.
```suggestion
  noReports,
  neverReported,
```